packages/cli/src/util/getHttpOperations.ts

```diff
@@ -0,0 +1,34 @@
+import { transformOas2Operation, transformOas3Operation } from '@stoplight/http-spec';
+import { parse } from '@stoplight/yaml';
+import * as fs from 'fs';
+import { defaults, flatten, get, keys, map } from 'lodash';
+import { httpAndFileResolver } from '../resolvers/http-and-file';
+
+export default async function getHttpOperations(spec: string) {
```

Comment:
This looks like it assumes the provided spec is always a local file and not a URL?

Reply:
You're right, there's a missing piece that I'm adding right now :)